database/factories/ServicePackageFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Service;
+use App\Models\Package;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\ServicePackage>
+ */
+class ServicePackageFactory extends Factory
```

Comment:
I usually do not generate factories for Pivot tables unless have more data stored in it, but we can leave it, it doesn't hurt haha